Jenkinsfile

```diff
@@ -33,6 +33,14 @@ pipeline {
                 python3 -m venv $TMPDIR
                 . $TMPDIR/bin/activate
                 pip install tox  # for tox supporting --parallel--safe-build
+
+                # Updates to allow building packages locally
+                sudo sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list
+                sudo apt-get update
+                sudo apt-get build-dep -y ubuntu-advantage-tools
+                sudo apt-get install -y --install-recommends sbuild ubuntu-dev-tools dh-systemd
+                # Missing build-deps
+                sudo apt-get install -y --install-recommends libapt-pkg-dev python3-mock python3-pytest
```

Comment:
I think we can skip this install steps? It's safe to assume that sbuild is installed on the Jenkins nodes and chroot setup for all the LTS releases, so we don't need to reinstall it. Also: if the package is built with sbuild we don't need to install any build-dep in the host system.

In general I think our goal should be to avoid `sudo` commands if possible, to ensure the CI run is safe and can't interfere too much with other things happening on the system.